_includes/sections/about.html

```diff
@@ -2,14 +2,16 @@
     <div class="container">
         <div class="row">
             <div class="col-md-8 col-md-offset-2 col-sm-12">
-                <h2>About Us</h2>
+                <h2>About</h2>
                 <hr>
-                <p class="lead">HackCU is a 24 hour hackathon organized annually at the University of Colorado Boulder.
-                    Every spring, we bring together people of all experience levels, majors and backgrounds to
-                    CU Boulder. The event provides the space and the resources for you to express your creativity,
-                    ingenuity, and determination to create anything you want to. We welcome any and all students to
-                    apply.</p>
+                <p class="lead">HackCU is all about learning, about improving yourself. 
+                    Spend a weekend with us at the University of Colorado Boulder. 
+                    We encourage students to take a step out of their comfort zone, to go beyond. 
```

Comment:
Can we make it more personal by saying `We encourage you to take a step of your comfort zone and go beyond`